__tests__/buildJs/pathToDotNotation.test.js

```diff
@@ -1,4 +1,4 @@
-const pathToDotNotation = require('../build').pathToDotNotation
+const pathToDotNotation = require('../../build').pathToDotNotation
```

Comment:
general feedback / question here, but could we start co-locating these tests to the source files? We do this in our other Primer repo's, and think it should apply here too.

Reply:
Those three tests will be removed once we move everything from `build.js` into the new build file.

So if we talk about colocation it would be for the config folder. I am not 100% sure I love the idea, as this either means we have double the files e.g. `16` instead of `8` which makes folders a bit hard to parse or, we need to put every transformer, etc. into a subfolder just to co-locate the test, which also makes it hard to work.

With components it feels more natural as you have the css module there as well.

Maybe we can discuss this in a call as this may be a bit of a longer conversation. 😉

Reply:
yeah, I can see why you'd want to keep these build.js specific tests here for now 👍 . I think the others _could_ move though. E.g. `__tests__/config/formats/typescript-export-definition.test.ts` can be colocated to `config/formats/typescript-export-definition.ts` and there's _less_ folder creation and traversal and those tests still pass fine. Might be missing something here, so happy to sync on it.